packages/synapse-react-client/test/lib/containers/table/datasets/DatasetItemsEditor.test.tsx

```diff
@@ -597,6 +603,7 @@ describe('Dataset Items Editor tests', () => {
       await userEvent.click(
         await screen.findByRole('button', { name: 'Cancel' }),
       )
+      await waitForElementToBeRemoved(dialog)
```

Comment:
Mui Dialog takes longer to close, so await its removal before querying screen for other 'dialog' elements